source/BaselineOfSuperluminal/BaselineOfSuperluminal.class.st

```diff
@@ -76,6 +87,12 @@ BaselineOfSuperluminal >> setUpPackages: spec [
 
 	spec
 		package: 'Superluminal-Service-Discovery-Tests'
-		with: [ spec requires: #( 'Superluminal-Service-Discovery' 'Hyperspace-SUnit' 'Teapot-Deployment') ];
-		group: 'Tests' with: 'Superluminal-Service-Discovery-Tests'
+		with: [ 
```

Comment:
Unrelated, but this formatting change is weird. Why lower the first block, when a dot was added after the last argument?

Reply:
It's a side effect of the Pharo 9 formatter... I cannot reproduce exactly the same configuration we had in Pharo 8 :shrug: